tests/Domain/PersonTest.php

```diff
@@ -63,6 +63,44 @@ public function testToMatchBotSummaryMessage(): void
         $this->assertEquals('cus_1234567890', $message->stripe_customer_id);
     }
 
+    public function testToStripeCustomerWithPasswordJustSet(): void
+    {
+        $person = $this->getPersonWithKeyFieldsSet();
+        $now = new \DateTimeImmutable('now');
+        $message = $person->getStripeCustomerParams(passwordAddedAt: $now);
+
+        $this->assertEquals([
+            'email' => $person->email_address,
+            'name' => $this->summariseName($person->first_name, $person->last_name),
```

Comment:
Any reason not to do 
```suggestion
            'name' => 'Loraine James',
```
or 
```suggestion
            'name' => self::CUSTOMER_NAME,
```
for simplicity instead of calling a function here and below?

Reply:
No, makes sense to simplify, thanks. Done in fec2d9aab76138f5106661d85a80cc9ed2317263